website/docs/r/domain.html.markdown

```diff
@@ -24,56 +24,34 @@ resource "libvirt_domain" "default" {
 The following arguments are supported:
 
 * `name` - (Required) A unique name for the resource, required by libvirt.
-   Changing this forces a new resource to be created.
-* `memory` - (Optional) The amount of memory in MiB. If not specified the domain will be
-   created with 512 MiB of memory
-   be used.
-* `vcpu` - (Optional) The amount of virtual CPUs. If not specified, a single CPU will be created.
-* `running` - (Optional) Use `false` to turn off the instance. If not specified, true is assumed and the instance, if stopped, will be started at next apply.
-* `disk` - (Optional) An array of one or more disks to attach to the domain. The `disk` object structure is documented below.
-* `network_interface` - (Optional) An array of one or more network interfaces to attach to the domain. The `network_interface` object structure is documented below.
+  Changing this forces a new resource to be created.
+* `cpu` - (Optional) Configures CPU mode. See [below](#cpu-mode) for more
+  details.
+* `vcpu` - (Optional) The amount of virtual CPUs. If not specified, a single CPU
+  will be created.
+* `memory` - (Optional) The amount of memory in MiB. If not specified the domain
+  will be created with 512 MiB of memory be used.
+* `running` - (Optional) Use `false` to turn off the instance. If not specified,
+  true is assumed and the instance, if stopped, will be started at next apply.
+* `disk` - (Optional) An array of one or more disks to attach to the domain. The
+  `disk` object structure is documented [below](#handling-disks).
+* `network_interface` - (Optional) An array of one or more network interfaces to
+  attach to the domain. The `network_interface` object structure is documented
+  [below](#handling-network-interfaces).
 * `metadata` - (Optional) A string containing arbitrary data. This is going to be
```

Comment:
I checked their project, this is not being used. I'm going to drop it from the docs.